config/locales/fr.yml

```
@@ -252,10 +255,21 @@ fr:
     not_used: Non utilisé
     ontology_not_valid: 'l''ontologie n''est pas valide, voici les erreurs : '
+    profile:
+      contributed_in: A un role dans
+      collaborated_with: A collaboré avec
+      collaborated_on: Ont collaboré sur
+      affiliated_agents: Agents affiliés
+      metadata: Métadonnées
+      statistics: Statistiques
+      no_contributions: Cet agent n'a aucune contribution dans %{portalName} 
+      keywords: Mots-clés
+      tags : Tags
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)